src/components/SliderInput.tsx

```diff
@@ -57,7 +57,7 @@ export const SliderInput = ({ min, max, prefixLabel, suffixLabel, name, step = 1
         <Slider
           tooltipAlwaysVisible={false}
           css={styles.slider}
-          min={min}
+          min={0}
```

Comment:
should this be `min ?? 0` or something? What about for probe frequency when the minimum value is 10 seconds?

Reply:
Yes good catch, this was for testing IIRC